scipy/sparse/tests/test_csc.py

```diff
@@ -35,5 +36,26 @@ def test_csc_getcol():
         assert_(type(csc_col) is csc_matrix)
 
 
-if __name__ == "__main__":
+def test_csc_nonzero():
+    A = np.array([[1, 0, 1], [0, 1, 1], [0, 0, 1]])
+    Asp = csc_matrix(A)
+
+    A_nz = set([tuple(ij) for ij in np.transpose(A.nonzero())])
+    Asp_nz = set([tuple(ij) for ij in np.transpose(Asp.nonzero())])
+
+    assert_equal(A_nz, Asp_nz)
+
+
+def test_numpy_csc_nonzero():
+    # See gh-5987
+    A = np.array([[1, 0, 1], [0, 1, 1], [0, 0, 1]])
+    Asp = csc_matrix(A)
+
+    A_nz = set([tuple(ij) for ij in np.transpose(np.nonzero(A))])
+    Asp_nz = set([tuple(ij) for ij in np.transpose(np.nonzero(Asp))])
+
+    assert_equal(A_nz, Asp_nz)
+
+
+if __name__ == "q__main__":
```

Comment:
This looks like a typo, but I think the changes to this file aren't needed. The tests in test_base.py should be sufficient.